modules/misc/news.nix

```
@@ -2201,6 +2201,17 @@ in {
           services into a video player.
         '';
       }
+
+      {
+        time = "2025-03-31T16:39:41+00:00";
+        condition = config.programs.jq.enable;
+        message = ''
+          Jq module now supports color for object keys
+
+          Your configuration will break if you have "programs.jq.colors" option.
+          To resolve this, please add `objectKeys` to your assignment of `programs.jq.colors`.
```

Sorry, missed it in the last round.

```suggestion
          Your configuration will break if you have defined the "programs.jq.colors" option.
          To resolve this, please add `objectKeys` to your assignment of `programs.jq.colors`.
```